### PR TITLE
1922 Add danish icons & fix bug with invisible nav title

### PR DESCRIPTION
**Pull request by zamorastp**

### CODE REVIEW

**Comment by zamorastp** [comment]
File

File: Sources/AccountSDKIOSWeb/Simplified Login/Model/DataModels.swift [hunk 1]
```diff
@@ -31,7 +31,9 @@ extension SimplifiedLoginNamedImageData {
             orderedIconNames = ["Finn", "VG", "Aftenposten", "E24", "BergensTidene"]
         case .proFi:
             orderedIconNames = ["Tori", "Oikotie", "Hintaopas", "Lendo", "Rakentaja"]
-        case .proDk, .pre:
+        case .proDk:
+            orderedIconNames = ["Dba", "Bilbasen"]
```
Comment (Daniel-Echegaray) [inline Sources/AccountSDKIOSWeb/Simplified Login/Model/DataModels.swift:35]: The assets names are "dba" and "bilbasen". With no capital letters.
You should probably change the asset names to have it working

**Review by Daniel-Echegaray — APPROVED** [review]
Looks good! Just update the icon names

File: Sources/AccountSDKIOSWeb/Simplified Login/Model/DataModels.swift (hunk 1 above)
Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/Simplified Login/Model/DataModels.swift]: Good catch, but only the names of directories matters. Fixed for dba.

Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/Simplified Login/Model/DataModels.swift]: Screenshot 2022-01-10 at 14 05 36

**Pull request merged**